Sources/SyncDataProviders/Bookmarks/internal/BookmarksResponseHandler.swift

```diff
@@ -121,15 +124,6 @@ final class BookmarksResponseHandler {
                 return
             }
 
-//            guard let favoritesUUIDs else {
-//                return
-//            }
-//            
-//            guard let favoritesFolder = BookmarkUtils.fetchFavoritesFolder(context) else {
-//                // Error - unable to process favorites
-//                return
-//            }
```

Comment:
What's with this? :)

Reply:
It's a remainder from pre-form-factor-specific favorites code which I forgot to remove :)